CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## 1.27.0
 
+* Adds an overload to `map.merge()` that supports merging a nested map.
+
+  `map.merge($map1, $args...)`: `$args` contains multiple keys, followed by a
```

Comment:
I'd write this as `map.merge($map1, $keys..., $map2)`. Even though `$map2` can't *actually* be passed by name, in informal documentation like this it'll help make it clearer what the intended usage is.